INSTRUMENTS/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ["KEITHLEY2400", "LABJACKU3","PHIDGET22"]
```

Comment:
Here we can import files from all category folders like: `from .KEITHLEY.KEITHLEY2400 import KEITHLEY2400`.